crates/common/tedge_utils/src/tedge_derive.rs

```diff
@@ -0,0 +1 @@
+pub use tedge_derive::serde_other;
```

Comment:
I don't understand why this module with a single `pub use` is useful.

Why not in `lib.rs`:

```rust
#[cfg(feature = "tedge-derive")]
pub use tedge_derive::serde_other;
```

Reply:
The updated version is clearer (with an inline module). Thanks.